coordinator/bridge/src/main/java/io/stargate/bridge/service/QueryHandler.java

```diff
@@ -122,6 +122,7 @@ protected CompletionStage<BridgeService.ResponseAndTraceId> buildResponse(Result
           Result.Rows rows = (Result.Rows) result;
 
           if (enrichResponse) {
+            // TODO raw use of column. Is this correct?
             RowDecorator rowDecorator =
                 connection.makeRowDecorator(TableName.of(rows.resultMetadata.columns));
```

Comment:
Tracking down one of the uses, it looks like we need to adjust this or we could hit the exception here:

https://github.com/stargate/stargate/blob/8c1208a461e68e3068fadb576f63ac812d78a1bf/coordinator/persistence-api/src/main/java/io/stargate/db/AbstractRowDecorator.java#L35-L52